setup.py

```diff
@@ -8,20 +8,28 @@
 from setuptools import setup, find_packages
 
 
-def get_version():
+def load_version():
     """Returns the version found in nilearn/version.py
 
     Importing nilearn is not an option because there may dependencies
     like nibabel which are not installed and setup.py is supposed to
     install them.
     """
-    locals_dict = {}
+    # load all vars into globals, otherwise
+    #   the later function call using global vars doesn't work.
     globals_dict = {}
     with open(os.path.join('nilearn', 'version.py')) as fp:
-        exec(fp.read(), globals_dict, locals_dict)
+        exec(fp.read(), globals_dict)
 
-    return locals_dict['__version__']
+    return globals_dict
 
+
+def is_installing():
+    # Allow command-lines such as "python setup.py build install"
+    return len(sys.argv) > 1 and 'install' in sys.argv
```

Comment:
Note that it doesn't matter in the end:

    Even for projects that do choose to use distutils, when pip
    installs such projects directly from source (rather than installing
    from a prebuilt wheel file), it will actually build your project using
    setuptools instead.